[All Hosts] (debugging) Preference for the debugging tool vs. network share deployment (#4655)
OfficeDev/office-js-docs-pr: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md

> Deployment by network share isn't supported for production add-ins. This method has the following limitations.
>
> - The add-in can only be installed on Windows computers.

> > - Add-ins that use the [unified manifest for Microsoft 365](../develop/unified-manifest-overview.md) are not supported by publishing to a network share.

Reviewer: ```suggestion
> - Add-ins that use the [unified manifest for Microsoft 365](../develop/unified-manifest-overview.md) aren't supported when published to a network share.
```
